eng/Versions.props

```diff
@@ -6,8 +6,10 @@
     <PreReleaseVersionIteration>7</PreReleaseVersionIteration>
     <!-- when bumping this you also need to rename the .pkgproj's in eng/nuget and eng/nuget/descriptions.json. TODO: make this better -->
```

Comment:
this comment needs to be updated:

```suggestion
    <!-- when bumping this you also need to update eng/nuget/descriptions.json. TODO: make this better -->
```

Or can we get rid of descriptions.json too?

Reply:
I think we can get rid of it